examples/dockerfiles/Kogito_swf.dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM quay.io/mdessi/kogito-swf-builder:latest AS builder
```

Comment:
```suggestion
FROM quay.io/kiegroup/kogito-swf-builder:latest AS builder
```

Reply:
This file is there to have a version to run within 50 sec instead 10 minutes

Reply:
so waiting for new 1.33 release ? :D

Reply:
removed due to https://issues.redhat.com/browse/KOGITO-8510